content/en/docs/started/workstation/getting-started-multipass.md

```diff
@@ -22,39 +22,66 @@ Here's a summary of the steps involved:
 1. Set up Microk8s
 2. Enable Kubeflow
 
-**Note:** the minimum version of Microk8s needed to enable Kubeflow is 1.18
+**Note:** the minimum version of Microk8s needed to enable Kubeflow is 1.18.
 
-### 1. Install and set up Microk8s
+### 1. Install Microk8s
```

Comment:
Unordered lists should not be used for ordered instructions. Consider revising these instructions to use ordered lists or combine the headings and content into one list. For example:

1.  Install MicroK8s with Snap by running the following command:

    ```
    sudo snap install microk8s --classic
    ```

1.  Verify that MicroK8s is running with the command:

    ```
    microk8s.status --wait-ready
    ```

1.  Enable common services on your Microk8s deployment:

    ```
    microk8s.enable dns dashboard storage
    ```

1.  Optional, to enable GPU support (available only for NVIDIA GPU hardware), run:  `microk8s.enable gpu`

1.  Deploy Kubeflow with the command:

    ```
    microk8s.enable kubeflow
    ```

    The deployment process may take a few minutes. Once completed, the script will print out the port number and 
    credentials to access the Kubeflow dashboard.

Reply:
This looks great! Thank you for this. I will update this in this PR so that it is backported to V1.0 and create a new PR for v1.1.